.scalafmt.conf

```diff
@@ -0,0 +1,20 @@
+align = true
+assumeStandardLibraryStripMargin = true
+maxColumn = 100
+
+# For better code clarity
+danglingParentheses = true
+
+docstrings = JavaDoc
+
+rewrite.rules = [
+  AsciiSortImports,
+  AvoidInfix,
+]
```

Comment:
In akka-discovery service locator, I have added the following:

```
rewrite.rules = [ 
  RedundantBraces, 
  ExpandImportSelectors, 
  RedundantParens,  
  SortModifiers, 
  PreferCurlyFors 
]
```

Not that I have strong opinions on this, but maybe worth review the rules. 

I particularly like:
* https://scalameta.org/scalafmt/docs/configuration.html#redundantbraces
* https://scalameta.org/scalafmt/docs/configuration.html#expandimportselectors - make it easier for merges and cherry-picks. Less conflicts.
* https://scalameta.org/scalafmt/docs/configuration.html#redundantparens

Reply:
`RedundantBraces` isn't idempotent, and I've come around to

```scala
def foo = {
  List(1, 2, 3).sum
}
```

being better.

* `ExpandImportSelectors`, sure, though it means dropping the auto-sort
* `RedundantParens` hmm, sure
* `PreferCurlyFors` I don't feel strongly for or against, so sure
* `SortModifiers` sure, but with another ordering